python_src/src/lamp_py/postgres/postgres_schema.py

```diff
@@ -247,7 +265,15 @@ class StaticRoutes(SqlBase):  # pylint: disable=too-few-public-methods
     route_sort_order = sa.Column(sa.Integer, nullable=False)
     route_fare_class = sa.Column(sa.String(30), nullable=False)
     line_id = sa.Column(sa.String(30), nullable=True)
-    static_version_key = sa.Column(sa.Integer, nullable=False, index=True)
+    static_version_key = sa.Column(sa.Integer, nullable=False)
+
+
+sa.Index(
+    "ix_static_routes_composite_1",
+    StaticRoutes.static_version_key,
+    StaticRoutes.route_id,
+    StaticRoutes.route_type,
+)
```

Comment:
We should have at least one index that starts with `static_version_key` on all static tables since that field is going to be used as a join key or where clause statement for just about any query involving them.